core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,34 +37,50 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+declare -A CONFIG_ENV
 # shellcheck disable=1091
 source /etc/trino/env.sh
 
+join_by() {
+    local IFS="$1"
+    shift
+    echo "$*"
+}
+
+sudo_args=(-u "$SERVICE_USER")
+for key in "${!CONFIG_ENV[@]}"; do
+    export "$key=${CONFIG_ENV[$key]}"
+done
+if [ "${#CONFIG_ENV[@]}" -ne 0 ]; then
+    sudo_args+=(--preserve-env="$(join_by , "${!CONFIG_ENV[@]}")")
+fi
+
 start() {
     echo "Starting ${SERVICE_NAME} "
     if [ -z "${JAVA_HOME}" ]; then
+        # shellcheck disable=2016
```

Comment:
Add comment why; should this part of the fmt commit?

Reply:
Done! Also moved to the fmt commit.